### ISSUE TITLE
Api is not working.

### ISSUE THREAD

**Issue by Amirden-st** [feature-sliced/tutorial-conduit#4]
API for this project doesn't work. Just try `npm run dev`. It gives Error 1016 by Cloudflare.

**Comment by bestwebdeveloper** [comment]
True. Unfortunately, realworld API is not maintained anymore https://github.com/gothinkster/realworld/issues/1611 😭.

**Comment by illright** [comment]
Thanks for bringing attention to this! I've developed a backend myself and integrated it into the tutorial code seamlessly, so it should now be possible to complete the tutorial again
